ui/src/views/Rules/ListRules/index.jsx

```diff
@@ -452,13 +452,24 @@ function ListRules(props) {
       const [productFilter, channelFilter] = productChannelQueries;
       const ruleProduct =
         rule.product || (rule.scheduledChange && rule.scheduledChange.product);
-      const ruleChannel =
-        rule.channel || (rule.scheduledChange && rule.scheduledChange.channel);
+      const ruleScheduledChange =
+        rule.scheduledChange && rule.scheduledChange.channel;
+      let ruleChannel;
 
       if (ruleProduct !== productFilter) {
         return false;
       }
 
+      if (ruleScheduledChange) {
+        if (ruleScheduledChange.includes('*')) {
```

Comment:
I don't think is right -- you want to check `ruleScheduledChange.channel` for this, not the entire rule.

Reply:
@bhearsum `ruleScheduledChange` returns just the channel not the entire rule. I have also tested it out as you indicated and it works as it should